### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by kevinforrestconnors**

### CODE REVIEW

**Comment by ghost** [comment]
sure!

File: app/assets/stylesheets/layout/navbar.css.scss [hunk 1]
```diff
@@ -0,0 +1,87 @@
+nav { 
+    color: white;
```
Comment (alexthemitchell) [inline app/assets/stylesheets/layout/navbar.css.scss:2]: Should you turn this into a variable as in line 3?

File: app/assets/stylesheets/layout/navbar.css.scss [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    position: relative;
+    top: 7px;
+    left: 4px;
+}
+
+.nav-selected { // this highlights the nav element linking to the current page
+    font-weight: bold;
+}
+
+.nav-li { // this describes a elements or li elements  or .nav-submenus
```
Comment (alexthemitchell) [inline app/assets/stylesheets/layout/navbar.css.scss]: `<a>` and `,<li>` makes a little more sense. I almost gave a grammatical correction for "a elements"

File: app/controllers/user_sessions_controller.rb [hunk 3]
```diff
@@ -11,7 +11,7 @@ def create
     if @user_session.save
       flash[:notice] = 'Login successful!'
       puts @current_user
-      redirect_back_or_default '/students' # account_url(@current_user)
+      redirect_back_or_default '/text' # account_url(@current_user)
```
Comment (alexthemitchell) [inline app/controllers/user_sessions_controller.rb]: What and why is this comment?

File: app/assets/stylesheets/layout/navbar.css.scss (hunk 1 above)
Comment (kevinforrestconnors) [inline app/assets/stylesheets/layout/navbar.css.scss:2]: white is a standard web color meaning rgb(255,255,255), I don't think turning it into a variable adds clarity, rather, it detracts clarity as I could assign whatever value I want to $white

**Comment by alexthemitchell** [comment]
Just a few small changes, but otherwise LGTM.

**Comment by alexthemitchell** [comment]
Also, in the future, make a new branch instead of reopening pull requests.

File: app/controllers/user_sessions_controller.rb (hunk 3 above)
Comment (kevinforrestconnors) [inline app/controllers/user_sessions_controller.rb]: I don't know.  I didn't add that comment

Comment (alexthemitchell) [inline app/controllers/user_sessions_controller.rb]: Remove it.

**Comment by kevinforrestconnors** [comment]
I reopened it because Ezra closed it instead of merging it :-1:

**Pull request merged**